gpu/kinfu_large_scale/tools/kinfu_app_sim.cpp

```diff
@@ -1096,13 +1099,16 @@ struct KinFuApp
 	
 	tic_toc.push_back (getTime ());
 
+  {
 	  SampledScopeTime fps(time_ms, i);
 	  //run kinfu algorithm
     bool has_image = (integrate_colors_) ? kinfu_ (depth_device_, image_view_.colors_device_) : kinfu_ (depth_device_);                  
 
+	}
+	
       }
       
-      tic_toc.push_back (getTime ());
+       tic_toc.push_back (getTime ());
```

Comment:
Erm... formatting please

Reply:
Some of these files have very weird formatting btw, we should `clang-format` more modules.

Reply:
Go forth. The world (err `.dev/format.sh`) is your playground. I didn't add GPU and CUDA before because I didn't trust CI to catch any bad formatting caused due to cuda files. But soon we'll have a build of CUDA in the CI so we'll know when we break things